config/paths.js

```diff
@@ -34,7 +34,9 @@ function resolveApp(relativePath) {
 // config after eject: we're in ./config/
 module.exports = {
   appBuild: resolveApp('build'),
+  appFavico: resolveApp('src/favicon.ico'),
   appHtml: resolveApp('index.html'),
+  appIndexJs: resolveApp('src/index.js'),
```

Comment:
Let’s use this variable instead of `path.join(paths.appSrc, 'index')` in `webpack.config.dev` and `.prod` too.